src/components/control/icons.css

```diff
@@ -0,0 +1,12 @@
+@import '../../styles/spacing.css';
+
+.container {
+  align-items: center;
+  display: grid;
+  grid-auto-flow: column;
+  grid-gap: var(--spacing-small);
+  position: absolute;
+  right: var(--spacing-medium);
+  transform: translateY(-50%);
```

Comment:
Just curious, what is the transform for?

Reply:
This `.icons` class is the "container". The container is absolutely position, and its top-border is placed in the "vertical midway point" of the parent container. 

<img width="93" alt="image" src="https://user-images.githubusercontent.com/841084/153484120-7f418c36-33dc-419e-8226-8af2360833d7.png">

This transform then "tweaks" the icons container by pushing it up by 50% of its height. This means the icons container is now vertically centered as an absolutely positioned element. 

<img width="92" alt="image" src="https://user-images.githubusercontent.com/841084/153484176-8d090ee4-ceeb-44c3-a460-a404dfb44d39.png">